tests/util/WhateversInTheFridge.py

```diff
@@ -100,6 +100,7 @@ def readNormalizationRecord(self, runId: str, useLiteMode: bool, version: Option
             useLiteMode=useLiteMode,
             version=version,
             calculationParameters=self.calculationParameters_with_stateId("0xdeadbeef"),
+            normalizatinoCalibrantSamplePath="path/to/calibrant",
```

Comment:
Mis-spelling `normalizationCalibrationSamplePath` (and also check it at the _source_ in the `NormalizationRecord` declaration).